vizro-core/src/vizro/_themes/create_chart_templates.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import json
+import re
+import sys
+from pathlib import Path
+from typing import Optional
+
+from plotly import graph_objects as go
+from plotly.utils import PlotlyJSONEncoder
+
+from vizro._themes._color_values import COLORS
+from vizro._themes.common_values import create_template_common
+
+THEMES_FOLDER = Path(__file__).resolve().parent
+CSS_FILE = THEMES_FOLDER.parent / "static" / "css" / "vizro-bootstrap.min.css"
```

Comment:
Also no need to split each part out as a separate string - you can just do `x / "static/css/vizro-bootstrap.min.css"` and so long as `x` is a pathlib path then it will work.